lib/features/auth/view/login_view.dart

```diff
@@ -0,0 +1,116 @@
+// ignore_for_file: public_member_api_docs, sort_constructors_first
+part of 'auth_imports.dart';
+
+class LoginView extends ConsumerStatefulWidget {
+  const LoginView({super.key});
+  static route() => MaterialPageRoute(builder: (_) => LoginView());
+
+  @override
+  ConsumerState<LoginView> createState() => _LoginViewState();
+}
+
+class _LoginViewState extends ConsumerState<LoginView> {
+  final TextEditingController emailController = TextEditingController();
+  final TextEditingController passwordController = TextEditingController();
+
+  void signIn() {
+    ref.read(authControllerProvider.notifier).login(
+        email: emailController.text,
+        password: passwordController.text,
+        context: context);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final isLoading = ref.watch(authControllerProvider);
+    return isLoading
+        ? const LoadingPage()
+        : Scaffold(
```

Comment:
For readability sake, do this instead:

```suggestion
    if (isLoading) {
      return LoadingPage();
    }
    
    return Scaffold(
      ...
```